macaroonbakery/codec.py

```diff
@@ -8,9 +8,9 @@
 from nacl.encoding import Base64Encoder
 from nacl.public import Box, PublicKey
 
-import macaroonbakery
-from macaroonbakery import bakery
-from macaroonbakery.checkers.namespace import deserialize_namespace
+from macaroonbakery import BAKERY_V1, BAKERY_V2, BAKERY_V3
+from macaroonbakery.third_party import legacy_namespace, ThirdPartyCaveatInfo
+from macaroonbakery.checkers import deserialize_namespace
```

Comment:
Maybe just import macaroonbakery.checkers ?

Reply:
in a follow up pr